tests/containers/award/AwardContainer-test.jsx

```diff
@@ -365,105 +367,231 @@ const txnData = {
 
         // reset the spies
         parseAwardSpy.reset();
     });
 
     // parse transactions
     it('should parse the returned transactions and send to redux store', () => {
+        const expectedTxnData = [{
+            _jsid: "contract-transaction-3",
+            action_date: "10/1/2014",
+            action_type: null,
+            description: "EXPRESS REPORT PHARMACY PRIME VENDOR VA760PPVFY2015OCT",
+            federal_action_obligation: "$227,269,217",
+            id: 34779,
+            modification_number: "0",
+            type: null,
+            type_description: null
+        }];
+        // Mock the call and validate it matches the expected txn summary
+        const mockReduxAwardTxn = jest.fn((args) => {
+            expect(args).toEqual(expectedTxnData);
+        });
+
+        const mockAppendAwardTxn = jest.fn((args) => {
+            expect(args).toEqual(expectedTxnData);
+        });
+
+        const mockSetTxnMeta = jest.fn((args) => {
+            expect(args).toEqual(award.transactionMeta);
+        });
+
+        const mockUpdateTxnRenderHash = jest.fn((args) => {
+            expect(args).toEqual(undefined);
+        });
+
+        const mockUpdateTxnGroupHash = jest.fn((args) => {
+            expect(args).toEqual(undefined);
+        });
+
         // Set up container with mocked award action
         const awardContainer = shallow(
             <AwardContainer
                 params={parameters}
                 award={award}
-                setAwardTransactions={AwardActions.setAwardTransactions}
-                appendAwardTransactions={AwardActions.appendAwardTransactions}
-                setTransactionsMeta={AwardActions.setTransactionsMeta}
-                updateTransactionRenderHash={AwardActions.updateTransactionRenderHash} />);
+                appendAwardTransactions={mockAppendAwardTxn}
+                setTransactionsMeta={mockSetTxnMeta}
+                updateTransactionRenderHash={mockUpdateTxnRenderHash}
+                updateTransactionGroupHash={mockUpdateTxnGroupHash}
+                setAwardTransactions={mockReduxAwardTxn} />);
 
         const parseTxnSpy = sinon.spy(awardContainer.instance(),
             'parseTransactions');
 
-        // Parse award
-        awardContainer.instance().parseTransactions(txnData);
+        // Run function on instance of container
+        awardContainer.instance().parseTransactions(txnData, true);
 
         // checking that it ran
         expect(parseTxnSpy.callCount).toEqual(1);
+        expect(mockReduxAwardTxn).toHaveBeenCalledTimes(1);
 
         // reset the spies
         parseTxnSpy.reset();
     });
 
     // next transaction page
     it('should fetch the next page of available transactions', () => {
+        // set initial page
+        const initialTxnMeta = {
+            page: 1
+        };
+        // Mock the call and validate it does NOT the expected award page
+        const MockReduxAwardTxnMeta = jest.fn((args) => {
```

Comment:
Lowercase the `M` please.

Also I'm still not sure this test is actually testing anything. `nextTransactionPage()` is not supposed to do anything if we're currently on the last page. The `award` Redux value you're setting the container up with has a `totalPage` of 1, so incrementing the page won't have any effect.

For this test, you should modify the `totalPage` count to be more than one and the current `page` to be 1. Then, call `nextTransactionPage` and verify that the `page` number changed.

You should also write a second test that verifies nothing happens when `nextTransactionPage` is called and we are currently on the last page.

Because `nextTransactionPage` calls `fetchTransactions` in turn, I would mock the entire `fetchTransactions` rather than trying to implement another promise/API mock and just verify that the right page number argument is being passed in.